.github/workflows/release.yaml

```diff
@@ -233,26 +235,19 @@ jobs:
           tag: ${{ github.ref }}
           overwrite: true
 
-  # Build for MacOS x86_64.
-  build-for-macOS-intel:
-    runs-on: macOS-latest
+  # Build for MacOS x86_64 and Apple Silicon
+  build-for-macos:
+    runs-on: macos-latest
     needs: get-version
     env:
-      TARGET: x86_64-apple-darwin
-      PROFILE: release-native
-      PLATFORM: macOS-intel
-      VERSION: ${{ needs.get-version.outputs.version }}
-      CFLAGS: -fno-stack-check
-      MACOSX_DEPLOYMENT_TARGET: 10.9
-
+        # macOS 11.0 Big Sur is the first version to support universal binaries
+        MACOSX_DEPLOYMENT_TARGET: 11.0
+        PROFILE: release-native
+        VERSION: ${{ needs.get-version.outputs.version }}
     steps:
       - name: Set PACKAGE environment variable
         run: |
-          echo "PACKAGE=${{ env.PACKAGE_NAME }}-${{ env.VERSION }}-${{ env.PLATFORM }}" >> "${GITHUB_ENV}"
-
-      - name: Set OUT_DIR environment variable
-        run: |
-          echo "OUT_DIR=build/${{ env.PACKAGE }}/${{ env.PACKAGE_NAME }}.app/Contents/MacOS" >> "${GITHUB_ENV}"
+          "PACKAGE=${{ env.PACKAGE_NAME }}-${{ env.VERSION }}-${{ env.PLATFORM }}" | Out-File -FilePath "${Env:GITHUB_ENV}" -Append
```

Comment:
I'm curious. This is powershell syntax, but `linux` and `macos-latest` runners are said to use `bash`.

Reply:
🤷‍♂️ I don't know about shells, I just followed https://github.com/rparrett/pixie_wrangler/blob/main/.github/workflows/release.yaml#L175

Reply:
[`c4aa286` (#112)](https://github.com/TheBevyFlock/bevy-template/pull/112/commits/c4aa286b027bd2bd8b770ba3b9f955b9f0933eed)